Adds plot of the anime in episode page

SwagOtaku opened the pull request:
I've made changes to the following files that puts the plot of the anime on the episode page. 
It looks like this https://imgur.com/a/w50Gh .

I've also added commented code in NineAnimeBrowser.py which puts the plot on the list of animes (i.e. the page before the episode page). But this heavily slows down the addon, when generating the anime list, it takes up to 30 seconds for the anime page to load because it has to request each animes' episode page and extract the plot

That's why the anime summary is on the episode page because the 9anime website has it set up this way, so the plot is faster to extract and does not affect the load time of the episode page.

DxCx commented on resources/lib/NineAnimeBrowser.py:
```diff
@@ -158,7 +167,12 @@ def get_anime_episodes(self, anime_url, returnDirectory=False):
         server = servers[mostSources]
-        return map(lambda x: utils.allocate_item(x['name'], x['url'], returnDirectory, ''), server)
+        resp = self._get_request(self._to_url("/watch/%s" % anime_url))
+        match = re.compile('<div class="desc">(.+?)</div>').findall(resp)
```
please move regex to class top like others =)

DxCx commented on resources/lib/NineAnimeBrowser.py:
```diff
@@ -59,14 +61,21 @@ def _handle_paging(self, results, base_url, page):
-        return [utils.allocate_item(name, base_url % next_page, True, None)]
+        return [utils.allocate_item(name, base_url % next_page,'', True, None)]
+
+    #def _handle_plot(self, res):
```
change it to "_extract_anime_plot(self, url)"
and uncomment, it's not used but might be in the future ;)

DxCx commented on resources/lib/NineAnimeBrowser.py:
```diff
@@ -59,14 +61,21 @@ def _handle_paging(self, results, base_url, page):
                           self._ANIME_VIEW_ITEMS_RE.findall(results))
         all_results += self._handle_paging(results, base_plugin_url, page)
-        return all_results
+        return all_results   
```
clean this plz hehe;)

SwagOtaku commented:
Whoops forgot to add utils.py. Sorry, my bad. ;}

I updated NineAnimeBrowser.py with the changes you requested and also added the updated utils.py.

MenaceFtV commented:
You might want to add in some autoscrolling text for the plot summary; the plot summary is cut off on devices such as the Amazon Fire TV (Generation 3, apparently unrootable) running a build such as Titanium.

Would it be possible to use a third party for metadata such as myanimelist, instead of using whatever is on 9anime?

SwagOtaku commented:
Updated NineAnimeBrowser.py to fix image url extraction

SwagOtaku commented:
It's all good man and thank you for the amazing addon
